moda-contracts/.env.example

```diff
@@ -0,0 +1,11 @@
+EVM_RPC_URL=
+MODA_DEPLOYER_PRIVATE_KEY=
```

Comment:
I think Wagmi has some scripts that will deploy, then write file names with the chain id in the name. The content of the file is the address. This is nice because we can commit it to Git, but ignore the `development` version if the filename starts with `development*`, for example. This is better than manually adding addresses to an env file

Reply:
So it might be better to write deploy scripts with wagmi. It is worth taking a look to compare